docs/Makefile

```diff
@@ -15,7 +15,7 @@ clean:
 	find $(SOURCEDIR) -name "*.rst" | grep -v index | xargs rm -f
 
 apidoc:
-	sphinx-apidoc -f --templatedir=$(TEMPLATEDIR) -o $(SOURCEDIR) ../datasetinsights/ ../datasetinsights/stats/visualization ../datasetinsights/dashboard.py ../datasetinsights/datasets/protos
+	sphinx-apidoc -f --templatedir=$(TEMPLATEDIR) -o $(SOURCEDIR) -d 2 ../datasetinsights/ ../datasetinsights/stats/visualization ../datasetinsights/dashboard.py ../datasetinsights/datasets/protos
```

Comment:
Does `-d 2` solve build error for readthedocs?

Reply:
No. It simply reduce visual noise on the landing page. This will reduce nested items like this
<img width="763" alt="Screen Shot 2020-09-08 at 12 27 45 PM" src="https://user-images.githubusercontent.com/786669/92519385-bc201e80-f1ce-11ea-9228-9ce3ba5a6c8d.png">